src/Catel.MVVM/Catel.MVVM.Shared/Collections/Extensions/SuspensionModeExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="SuspensionModeExtensions.cs" company="Catel development team">
+//   Copyright (c) 2008 - 2017 Catel development team. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Catel.Collections.Extensions
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// The suspension mode extensions.
+    /// </summary>
+    internal static class SuspensionModeExtensions
+    {
+        #region Fields
+        /// <summary>
+        /// The mixed modes.
+        /// </summary>
+        private static readonly List<SuspensionMode> MixedModes;
+        #endregion Fields
+
+        #region Constructors
+        /// <summary>
+        /// Initializes static members of the <see cref="SuspensionModeExtensions"/> class.
+        /// </summary>
+        static SuspensionModeExtensions()
+        {
+            MixedModes = new List<SuspensionMode>
```

Comment:
That would be: Enum<SuspensionMode>.GetValues().Where(mode => mode.ToString().ContainsIgnoreCase("Mixed")).ToList();